data_model/src/trigger.rs

```diff
@@ -34,100 +34,50 @@ model! {
     #[display(fmt = "@@{id}")]
     #[getset(get = "pub")]
     #[ffi_type]
-    pub struct Trigger<F: Filter> {
+    pub struct Trigger<F: Filter, E = Executable> {
```

Comment:
yet nowhere do you use the implicit default

Reply:
Yeah, will remove it